packages/backend-defaults/src/entrypoints/rootHttpRouter/rootHttpRouterServiceFactory.ts

```diff
@@ -30,6 +32,26 @@ import {
 } from './http';
 import { DefaultRootHttpRouter } from './DefaultRootHttpRouter';
 import { createHealthRouter } from './createHealthRouter';
+import { createLifecycleMiddleware } from './createLifecycleMiddleware';
+
+export function getConfigInHumanDuration(
+  config: RootConfigService,
+  key: string,
+): HumanDuration | undefined {
+  const value = config.getOptional(key);
+  if (typeof value === 'undefined') {
+    return undefined;
+  }
+  if (typeof value === 'number') {
+    return { milliseconds: value };
+  }
+  if (typeof value === 'string') {
+    return {
+      milliseconds: parseInt(value, 10),
+    };
+  }
```

Comment:
I don't think we want to support these, should be enough with just the built-in functionality from `readDurationFromConfig`

Reply:
Oh sorry, I didn't realize that `readDurationFromConfig ` supports string, and in a better way than I implemented it 🤦🏻‍♀️ . Just fixed it.